### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by krishnapitike** [deepmodeling/deepmd-kit#1466] [Q&A]
I have successfully installed deepmd using easy conda install:
conda create -n deepmd deepmd-kit=*=*cpu libdeepmd=*=*cpu lammps-dp -c https://conda.deepmodeling.org

Now, do I have to follow the instructions to do "Install from source code" ? or is it redundant with the previous step?

Thanks for your help!

**Comment by njzjz** [comment]
It is redundant with the previous step.
